src/App.js

```diff
@@ -1,63 +1,22 @@
-import { SearchBar } from "./modules/recipes/search-bar";
 import "./App.css";
-import { useEffect, useState } from "react";
-import { fetchRecipesByIngredient } from "./modules/recipes/recipeService";
-import RecipeList from "./modules/recipes/recipe-list/RecipeList";
-// import { Banner } from "./modules/recipes/banner";
+import { AppRouter } from "./AppRouter";
+import { Navbar } from "./modules/shared-components/Navbar";
+import { Banner } from "./modules/recipes/banner/banner";
+import { RecipesProvider } from "./modules/recipes/RecipesProvider";
```

Comment:
Remove after removing RecipesProvider below, it's already added in src/index.js file.